go.mod

```diff
@@ -4,7 +4,7 @@ go 1.13
 
 require (
```

Comment:
Do all these dependencies need to be updated? Can we only add what's necessary for Helm?

Reply:
You mean the `replace` block?